docs/examples/contrib/sqlalchemy/sqlalchemy_async_repository.py

```diff
@@ -19,42 +19,43 @@
     AsyncSessionConfig,
     SQLAlchemyAsyncConfig,
     SQLAlchemyInitPlugin,
-    base,
-    filters,
-    repository,
 )
+from litestar.plugins.sqlalchemy.base import UUIDAuditBase, UUIDBase
+from litestar.plugins.sqlalchemy.filters import LimitOffset
+from litestar.plugins.sqlalchemy.repository import SQLAlchemyAsyncRepository
 
 if TYPE_CHECKING:
     from sqlalchemy.ext.asyncio import AsyncSession
 
 
 class BaseModel(_BaseModel):
-    """Extend Pydantic's BaseModel to enable ORM mode"""
+    """Extended Pydantic's `BaseModel` to enable the ORM mode."""
```

Comment:
The phrasing here is intentional, as it's meant to be instructive rather than descriptive. Although it should probably go in a regular comment not a docstring

Reply:
Yes, you are right. I did not pay attention that this is inside a documentation string, so it was quite fine as it were.